extension/ts/main/adapters/sheets/util.ts

```diff
@@ -0,0 +1,26 @@
+const incrementColumn = (column: string): string => {
+	const parts = column.split("");
+	const incParts = (parts: string[]): string[] => {
+		if (parts.length === 0) {
+			return ["A"];
+		} else {
+			const last = parts.pop();
+			if (last === "Z") {
+				return [...incParts(parts), "A"];
+			} else {
+				return [...parts, String.fromCharCode(last.charCodeAt(0) + 1)];
+			}
+		}
+	};
+	return incParts(parts).join("");
+};
+
+const incrementColumnBy = (column: string, n: number): string => {
+	if (n <= 0) {
+		return column;
+	} else {
+		return incrementColumnBy(incrementColumn(column), n - 1);
+	}
+};
```

Comment:
cool, but what was the motivation for this -- why do we only need it to support the tags use case

Reply:
is this cuz the tags live in several diff sheets

Reply:
ok I think I documented it
https://github.com/braxtonhall/library-thing/blob/8744fd485ce96b5be8bb8e967477524192eb197d/extension/ts/main/adapters/tags/getTags.ts#L10-L23